extensions/eda/plugins/event_source/kafka.py

```diff
@@ -129,6 +140,17 @@ async def main(  # pylint: disable=R0914
     logger = logging.getLogger()
 
     topic = args.get("topic")
+    topics = args.get("topics")
+    topic_pattern = args.get("topic_pattern")
```

Comment:
@bzwei It might make sense to call re compile on the string to see if its a valid regex.
```
import re
try:
        re.compile(topic_pattern)
except re.error:
       raise ValueError("Invalid regex " + topic_pattern + " error: str(re.error))
```